wasmcloud-wasi-kv/README.md

```diff
@@ -0,0 +1,125 @@
+# `wasmcloud-wasi-kv`
+
+This repository contains a [WebAssembly System Interface (WASI) key-value contract][wasi-kv] compliant Web Assembly module which can be composed into a "macro" or "composite" component to provide key value functionality.
```

Comment:
Are macro/composite components the official language?

Also nit: `Web Assembly` -> `WebAssembly`